tests/TestCase/Pdf/CakePdfTest.php

```diff
@@ -97,7 +97,7 @@ public function testCustomTemplatePaths($config)
         $pdf->template('view', 'default');
         $pdf->templatePath('Posts/pdf');
         $result = $pdf->output();
-        $expected = '<h2>Rendered with default layout</h2>' . "\n" . 'Post data: testing';
+        $expected = '<h2>Rendered with default layout</h2>' . "\r\n" . 'Post data: testing';
         $this->assertEquals($expected, $result);
```

Comment:
You can also use assertTextEquals() and dont have to change the line endings - will be cross OS safe.